packages/in_app_purchase/in_app_purchase/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.2.3
+* Bumps minimum `in_app_purchase_storekit` version to 0.4.0.
+* Bumps example `in_app_purchase_storekit` version to 0.4.0.
+
 ## 3.2.2
 
 * Updates `in_app_purchase_storekit` to 0.4.0 
```

Comment:
Let's delete this line as part of this PR. We minimize changing old entries, but when an old entry is actively wrong we do generally fix it up, since most people are seeing the changelog by looking at the current version of pub.dev, so fixing old entries is useful.